bears/haml/HamlLintBear.py

```diff
@@ -0,0 +1,367 @@
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
+    SEE_MORE = 'https://github.com/brigade/haml-lint'
+
+    @staticmethod
+    def generate_config(filename, file,
+                        alignment_tabs_enable: bool=True,
+                        alt_text_enable: bool=True,
+                        class_attribute_with_static_value_enable: bool=True,
+                        classes_before_ids_enable: bool=True,
+                        consecutive_comments_enable: bool=True,
+                        consecutive_comments_max: int=1,
+                        consecutive_silent_scripts_enable: bool=True,
+                        consecutive_silent_scripts_max: int=2,
+                        empty_object_reference_enable: bool=True,
+                        empty_script_enable: bool=True,
+                        final_newline_enable: bool=True,
+                        final_newline_present: bool=True,
+                        html_attributes_enable: bool=True,
+                        id_names_enable: bool=True,
+                        id_names_style: str='lisp_case',
+                        implicit_div_enable: bool=True,
+                        indentation_enable: bool=True,
+                        indentation_character: str='space',
+                        indentation_width: int=2,
+                        inline_styles_enable: bool=True,
+                        instance_variables_enable: bool=True,
+                        instance_variables_file_types: str='partials',
+                        instance_variables_matchers_all: str='.*',
+                        instance_variables_matchers_partials:
+                        str='\A_.*\.haml\z',
+                        leading_comment_space_enable: bool=True,
+                        line_length_enable: bool=True,
+                        line_length_max: int=80,
+                        multiline_pipe_enable: bool=True,
+                        multiline_script_enable: bool=True,
+                        object_reference_attributes_enable: bool=True,
+                        repeat_id_enable: bool=True,
+                        repeat_id_severity: str='error',
+                        rubo_cop_enable: bool=True,
+                        rubo_cop_ignored_cops: typed_list(str) = (),
+                        ruby_comments_enable: bool=True,
+                        space_before_script_enable: bool=True,
+                        space_inside_hash_attributes_enable: bool=True,
+                        space_inside_hash_attributes_style: str='space',
+                        tag_name_enable: bool=True,
+                        trailing_whitespace_enable: bool=True,
+                        unnecessary_interpolation_enable: bool=True,
+                        unnecessary_string_output_enable: bool=True,
+                        view_length_enable: bool=True,
+                        view_length_max: int=100):
```

Comment:
@Makman2 @jayvdb Hmm... Remembering discussion in https://github.com/coala/coala-bears/pull/1967 about naming conventions... Even if not done dynamically like I did in the latter PR, the above setting names are also just directly taken over from the linter tool... What do you think about this case?

Reply:
if possible try to match the existing names from other bears that do the same. If there're no such settings, it can be kept.
About your dynamic bear settings thing: It's not a bad idea, however this is easier and works directly. Also we maintain for now a very fixed and stable API this way :+1: We can still adapt to your dynamic approach later :)